src/accordion/index.js

```diff
@@ -24,7 +24,7 @@ class Accordion extends Component {
     sections:            PropTypes.array.isRequired,
     uniqueIdentifier:    PropTypes.string,
     initialSectionIndex: validateIndex,
-    openSectionIndex:    validateIndex
+    openSectionIndex:    validateIndex,
```

Comment:
Dealt with by Babel. This is on purpose actually, as it means easier reordering of properties.